src/python/grpcio/grpc/_channel.py

```diff
@@ -843,17 +843,11 @@ def _unsubscribe(state, callback):
 
 def _options(options):
   if options is None:
```

Comment:
Favor
```
if <condition>:
  return <first expression>
else:
  return <second expression>
```
over
```
if <condition>:
  <name> = <first expression>
else:
  <name> = <second expression>
return <name>
```
.

(Also avoid reassigning behavior parameters when you can.)